services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/verify_email.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import qs from 'qs'
 
 import AssignActivityPackBanner from '../assignActivityPackBanner'
-import { Snackbar, defaultSnackbarTimeout, Spinner, } from '../../../../Shared/index'
+import { Snackbar, defaultSnackbarTimeout, Spinner, ADMIN, TEACHER, } from '../../../../Shared/index'
```

Comment:
I see the import of `TEACHER` here, but no references.  It's entirely possible that it's being used but Github isn't showing it, but I figured I'd double-check.